pallets/system/Cargo.toml

```diff
@@ -54,24 +55,28 @@ std = [
 	"frame-support/std",
 	"log/std",
 	"scale-info/std",
-	"serde",
+	"serde/std",
 	"sp-core/std",
 	"sp-io/std",
 	"sp-runtime-interface/std",
 	"sp-runtime/std",
 	"sp-std/std",
 	"sp-version/std",
 	"sp-weights/std",
-	"beefy-merkle-tree/std",
+	"binary-merkle-tree/std",
 ]
 runtime-benchmarks = [
 	"frame-support/runtime-benchmarks",
 	"sp-runtime/runtime-benchmarks",
+	"serde",
 ]
 try-runtime = [
 	"frame-support/try-runtime",
 	"avail-core/try-runtime"
 ]
+serde = [
+	"dep:serde",
```

Comment:
This is something new, what does it do?

Reply:
Have given the context [here](https://github.com/availproject/engineering/issues/249)